src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public Figure getRandomFigure() {
+        Random random = new Random();
+        ColorSupplier colorSupplier = new ColorSupplier();
+        int randNum = random.nextInt(5);
```

Comment:
5 is a magic number
https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced
![image](https://user-images.githubusercontent.com/20326858/209122957-645577da-c13e-4e0d-b8d9-ed1ceb673fd3.png)